GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2134,6 +2134,16 @@ private void QuickFetch()
             UICommands.RepoChangedNotifier.Notify();
         }
 
+        protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
+        {
+            return !IsDesignMode && HotkeysEnabled
+                && (base.ProcessCmdKey(ref msg, keyData) // upstream
```

Comment:
Handle downstrem first?
Generic

Reply:
The sequence is:
- current control
- parents (form)
- new: other visible controls of a form which supports it

Perhaps, I should explain "upstream" / "downstream" in the comment.
Would "upstream (parent form)" suffice?

Reply:
parents form (upstream) is better I believe